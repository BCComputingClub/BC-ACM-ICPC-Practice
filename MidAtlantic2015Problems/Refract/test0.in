600 600 1000 1.333 1.01
600 1200 4000 1.5 1.01
0 0 0 0 0
