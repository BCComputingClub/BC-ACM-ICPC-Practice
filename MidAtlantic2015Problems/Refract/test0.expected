44.37
11.51
